src/components/Formulários/Formulario.vue

```diff
@@ -0,0 +1,369 @@
+                this.$emit("nextStep");
+            },
+        }
+    }
+</script>
+
+<style scoped>
+    .main {
+        height: 100%;
+    }
+    
+    .form {
+        display: flex;
+        justify-content: center;
+    }
+
+    form {
+        width: 60%;
+    }
+
+    .welcome {
+        display: flex;
+        flex-direction: column;
+        width: 60%;
+        margin: 0 auto;
+    }
+
+    .input-box {
+        display: flex;
+        flex-direction: column;
+        position: relative;
+        width: 100%;
+    }
+
+    .input-box label {
+        padding: 0 .35rem;
+        margin-bottom: .45rem;
+        font-size: .80rem;
+        font-weight: 700;
+        background-color: #f5f5f5;
+        color: rgba(0, 0, 0, .7);
+        top: -.50em;
+        left: 2em;
+        z-index: 1;
+        text-transform: uppercase;
+        position: absolute;
+    }
+
+    .input-box input {
+        padding: .50rem .5rem;
+        background-color: #f5f5f5;
+        border: 1px solid rgba(0, 0, 0, 0.7);
+        border-radius: 5px;
+        outline: none;
+        letter-spacing: 1px;
+        line-height: 6ex;
+        position: relative;
+    }
+
+    input:hover {
+        background: #e2eeee;
+        transition: 200ms;
+    }
+
+    .input-columns {
+        display: flex;
+        gap: 1rem;
+        margin: .80rem 0;
+    }
+
+    .email-area,
+    .input-date {
+        position: relative;
+    }
+
+    .email-area input {
+        padding: .50rem 2.5rem;
+        
+    }
+
+    .email-icon {
+        position: absolute;
+        top: 1.1rem;
+        left: .70rem;
+        width: 1.5rem;
+    }
+
+    .input-date label {
+        padding: 0 .35rem;
+        font-size: .70rem;
+        top: -.50em;
+        left: 1em;
+        z-index: 1;
+        background-color: #f5f5f5;
+        position: absolute;
+    }
+
+    .input-date input {
+        display: flex;
+        padding: 0 .50rem;
+        width: 49%;
+        border-radius: 5px;
+        outline: none;
+        position: relative;
+        line-height: 6ex;
+        background-color: #f5f5f5;
+        border: 1px solid rgba(0, 0, 0, 0.7);
+        position: relative;
+    }
+
+    .checkbox-area {
+        display: flex;
+        gap: 2rem;
+        width: 37%;
+    }
+
+    h1 {
+        font-size: 2.8rem;
+        color: var(--main-dark-opacity-08);
+        transition: 200ms;
+    }
+
+    h1:hover {
+        color: #221e1ecc;
+        transition: 200ms;
+    }
+
+    p {
+        max-width: 500px;
+        font-size: 1rem;
+        font-weight: 500;
+        text-align: justify;
+        margin: 1rem 0;
+        color: var(--main-dark-opacity-08);
+    }
+
+    span {
+        font-weight: 700;
+        color: var(--main-magenta);
+    }
+
+    button {
+        margin-bottom: 1rem;
+        align-self: flex-start;
+        padding: 1rem 3rem;
+        background: var(--main-magenta);
+        box-shadow: 0px 0px 5px rgba(0, 0, 0, .2);
```

Comment:
Poderia remover o px
```suggestion
        box-shadow: 0 0 5px rgba(0, 0, 0, .2);
```